packages/contracts/contracts/core/ManagedService.sol

```diff
@@ -45,13 +51,13 @@ contract ManagedService is AccessControl, ReentrancyGuard {
      * @return bytes The return value of the underlying call.
      */
     function exec(
-        address _to,
+        address payable _to,
         bytes calldata _data
     ) public payable nonReentrant returns (bytes memory) {
         require(hasRole(RELAYER_ROLE, msg.sender), "ManagedService: invalid caller");
         require(_to != address(0), "ManagedService: target is address(0)");
 
-        emit Called(msg.sender, _to, keccak256(_data));
+        emit Called(msg.sender, _to, msg.value, keccak256(_data));
 
         // slither-disable-next-line arbitrary-send-eth
         (bool success, bytes memory res) = _to.call{ value: msg.value }(_data);
```

Comment:
Nit: You might already have thought of this, but we could also just cast `_to` as `payable` within the function. That might make calling this function slightly nicer. It's no big deal though I'm good with either. 
```suggestion
        (bool success, bytes memory res) = payable(_to).call{ value: msg.value }(_data);
```

Reply:
I don't have a preference. I'll just keep it as-is